README.md

```diff
@@ -4,7 +4,7 @@
 
 > *Nintendo Switch homebrew multitool, for several purposes and with many, many features!*
 
-- **If you are looking for Tinfoil, this is Tinfoil's safer and way more extended evolution.**
+- **If you are looking for Adubbz Tinfoil, this is it's more extended evolution.**
```

Comment:
Grammar: `it's` should be `its`.

Reply:
This sentence has little value. If this is a fork, perhaps something like this makes more sense:
`This is a fork of [Adubbz's Tinfoil](https://github.com/Adubbz/Tinfoil)`

Reply:
This is just a joke PR, to see if XorTroll would actually specify which Tinfoil he was on about due to his drama with the Lithium developer.
Thanks for the review, but I will be closing this now since there is no real need for it to stay anymore.